workflows-scripts/users/users-loan-renewal/README.md

```diff
@@ -18,4 +18,19 @@
 ##### Frontend:
 - folio_users-4.0.6
 
-- FOLIO release: Goldenrod
\ No newline at end of file
+- FOLIO release: Goldenrod
+
+As per requirements in JIRA, script will run against 
+- 10 loans (1, 5, 10 users)
+- 50 loans (1, 5, 10 users)
+- 100 loans (1, 5, 10 users)
+- 200 loans (1, 5, 10 users)
+
+## Setup data before running JMeter script:
+- checkin-checkout-db-restore.sql:
+This is the same script from check-in-check-out. It restores the DB to its original state. This script should be run after each test run to ensure that subsequent test runs have the same starting point. This script truncates all loans and requests and their associated tables, and also restores all the inventory items' status to "Available".
+
+`psql -h gcp1-db-01.c5cqgppfbl8c.us-east-1.rds.amazonaws.com -d folio -U folio -f checkin-checkout-db-restore.sql -a --echo-all`
+
+- Create open loans for user:
+`psql -h gcp1-db-01.c5cqgppfbl8c.us-east-1.rds.amazonaws.com -d folio -U folio -a -c "\copy fs09000000_mod_circulation_storage.loan(id, jsonb) FROM loans_5.tsv DELIMITER E'\t'"`
```

Comment:
Please remove the credentials values from the commands!

Reply:
We don't have credentials anywhere.